### ISSUE TITLE
Pluggin do not work in VSCode v1.21.0

### ISSUE THREAD

**Issue by lpreterite** [mjbvz/vscode-markdown-mermaid#15]
```
# VSCode

版本 1.21.0
提交 9a199d77c82fcb82f39c68bb33c614af01c111ba
日期 2018-03-07T11:04:09.886Z
Shell 1.7.9
渲染器 58.0.3029.110
Node 7.9.0
架构 ia32
```
Has been confirmed when version 1.19.0 is available, when the update to 1.21.0 plugin fails.

**Comment by riberon** [comment]
Version 1.21.0
Commit 9a199d77c82fcb82f39c68bb33c614af01c111ba
Date 2018-03-07T11:04:09.969Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64

**Comment by mjbvz** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/45007

Should be fixed in VS Code 1.21.1
